edge-rtac/src/main/resources/domain/edge-rtac/rtac-junit.feature

```diff
@@ -0,0 +1,20 @@
+Feature: edge-rtac integration tests
+
+  Background:
+    * url baseUrl
+    * table modules
+      | name                      |
+      | 'mod-login'               |
+      | 'mod-permissions'         |
+      | 'mod-inventory'           |
+      | 'mod-inventory-storage'   |
+      | 'edge-rtac'               |
+
+    * table adminAdditionalPermissions
+      | name                      |
+
+    * table userPermissions
+      | name                      |
+
+  Scenario: create tenant and users for testing
+    * call login { tenant: 'diku', name: 'diku_admin', password: 'admin' }
```

Comment:
Please add a comment describing why the diku (rather than tenant created for the Karate tests) is used

Reply:
Comment added.